src/main/java/seedu/homerce/logic/commands/revenue/BreakdownRevenueCommand.java

```diff
@@ -0,0 +1,112 @@
+package seedu.homerce.logic.commands.revenue;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.homerce.logic.parser.CliSyntax.PREFIX_MONTH_OF_YEAR;
+import static seedu.homerce.logic.parser.CliSyntax.PREFIX_YEAR;
+
+import java.time.Month;
+import java.time.Year;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import seedu.homerce.logic.commands.Command;
+import seedu.homerce.logic.commands.CommandResult;
+import seedu.homerce.model.HistoryManager;
+import seedu.homerce.model.Model;
+import seedu.homerce.model.revenue.Revenue;
+import seedu.homerce.model.revenue.predicate.RevenueMonthPredicate;
+import seedu.homerce.model.revenue.predicate.RevenueYearPredicate;
+
+/**
+ * Breaks down revenue by Service.
+ */
+public class BreakdownRevenueCommand extends Command {
+
+    public static final String COMMAND_WORD = "breakdownrev";
+    public static final String MESSAGE_SUCCESS = "Revenue breakdown: %1$s";
+    public static final String MESSAGE_USAGE = "Displays the breakdown of revenue for the month and year specified \n"
+            + "Parameters: " + PREFIX_MONTH_OF_YEAR + "MONTH " + PREFIX_YEAR + "YEAR \n" + "Example: " + COMMAND_WORD
+            + " " + PREFIX_MONTH_OF_YEAR + "12" + PREFIX_YEAR + "2020";
+
+    private final Month month;
+    private final Year year;
+
+    /**
+     * Constructor for breakdown of revenue.
+     *
+     * @param month month of revenue, value from 1-12
+     * @param year year of revenue, value > 0
+     */
+    public BreakdownRevenueCommand(Month month, Year year) {
+        this.month = month;
+        this.year = year;
+    }
+
+    @Override
+    public CommandResult execute(Model model, HistoryManager historyManager) {
+        requireNonNull(model);
+        List<Revenue> filteredRevenueList = createFilteredRevenue(model);
+        Map<String, Double> revenueMap = breakdownExpenses(filteredRevenueList);
```

Comment:
I think your method naming here should be `breakdownRevenue` instead of `breakdownExpenses`